types/json-schema-faker/tsconfig.json

```diff
@@ -0,0 +1,17 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "lib": ["es6"],
+        "noImplicitAny": true,
+        "strictNullChecks": true,
+        "esModuleInterop": true,
```

Comment:
You would probably have to remove that line, it's not enforced by linter, but anyway a rule, e.g.:
https://github.com/DefinitelyTyped/DefinitelyTyped/pull/44595#discussion_r425400276
```suggestion
```
so effectively, imports would have to be rewritten I think to:
```ts
import jsf = require('json-schema-faker');
import { Schema } from 'json-schema-faker';
```